.env.example

```diff
@@ -3,9 +3,11 @@ MYSQL_DATABASE=hackmd
 MYSQL_PASSWORD=
 MYSQL_ROOT_PASSWORD=
 
+HMD_DOMAIN=hackmd.example.com
+
 CMD_GOOGLE_CLIENTID=
 CMD_GOOGLE_CLIENTSECRET=
-CMD_GOOGLE_ALLOWEDDOMAINS=deepcompute.com,nudjur.com
+CMD_GOOGLE_ALLOWEDDOMAINS=example1.com,example2.com
```

Comment:
Why example domains?

Reply:
Since this is the .env.example I was just making it more generic. I can add back our own domains if you want.